src/system/RoutesModule/Form/Type/Base/AbstractRouteType.php

```diff
@@ -276,27 +276,25 @@ public function addReturnControlField(FormBuilderInterface $builder, array $opti
     /**
      * Adds submit buttons.
      *
-     * @param FormBuilderInterface $builder The form builder.
-     * @param array                $options The options.
+     * @param FormBuilderInterface $builder The form builder
+     * @param array                $options The options
      */
     public function addSubmitButtons(FormBuilderInterface $builder, array $options)
     {
         foreach ($options['actions'] as $action) {
             $builder->add($action['id'], 'Symfony\Component\Form\Extension\Core\Type\SubmitType', [
-                'label' => $this->__(/** @ignore */$action['title']),
+                'label' => $this->__($action['title']),
```

Comment:
The root problem of that is that workflow file's attributes (in the workflow xml definition) can not be translated. Any idea how to improve that?